pandas/tests/test_nanops.py

```diff
@@ -152,14 +149,13 @@ def check_fun_data(
         targfunc,
         testarval,
         targarval,
-        targarnanval,
```

Comment:
Can you just comment on what this removal is?

Reply:
On 212-213 we check if `targarnan is None` and if so set `targarnan = testar`.  That condition _always_ holds in the test, so  ripped it out, and targarnanval as a consequence